docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,56 @@
+
+## Introduction
+As any distributed service does, sometimes ecChronos breaks and you will have to troubleshoot what is going on. Generally speaking you can debug ecChronos like any other distributed Java program, meaning that you have to find which instances in your cluster are misbehaving and then isolate the problem using logs and tools. Luckily ecChronos had a great set of instrospection tools to help you.
+
+## Find The Misbehaving Instances
+The first step to troubleshooting a ecChronos issue is to use error messages, metrics and monitoring information to identify if the issue lies with the clients or the server and if it does lie with the server find the problematic nodes in the Cassandra cluster. The goal is to determine if this is a systemic issue (e.g. a problem pattern that affects the entire cluster) or isolated to a subset of instances (e.g. neighbors holding a lock or even a single node with bad hardware).
+
+## Client Logs and Errors
+During repair operations in a cassandra instance, triggered by ecChronos, it can return erros logs, like below:
+
+|Error                                                                   | Possible cause                                                                                     |
+|------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+|"Unexpected interrupt while trying to connect to Cassandra"             | Cassandra Node Overloaded, Bad Configuration on Installing, ecChronos Tables does not exists, etc. |
+|"Unable to register watch service, configuration refresh will not work" | Path Directory does not exists, File Permission disagred with request, etc.                        |
+|"Unable to get maxRepaired for {<table_reference>}"                     | Cassandra Node Overloaded, Instance Not Found, Table does not Exists Anymore, etc.                |
+
+*Futhermore, one common problem is when repair is taking too much time/is overdue. This can be caused by too many vnodes, nodes going down while holding a lock and Cassandra nodes being down.*
+
+## Metrics
+If you have Cassandra metrics reporting to a centralized location such as Graphite or Grafana you can typically use those to narrow down the problem. But ecChronos can be monitored, as described in the [repair monitoring documentation](REPAIR_MONITORING.md). Using metrics, it can be possible to identify some possible problems in the cluster and evaluete some basic things, like overload, network issues, cassandra rejections, etc.
```

Comment:
"But ecChronos can be..." -> "ecChronos can be ..."

"it can be possible to identify some possible problems" -> "it is possible to identify problems"

evaluete -> evaluate